symfony/plugins/orangehrmCorePlugin/Api/V2/Validator/Rules/Base64Attachment.php

```diff
@@ -77,17 +77,17 @@ public function validate($input): bool
             return false;
         }
 
+        $fileExtension = pathinfo($input['name'])['extension'];
```

Comment:
Are there any scenarios `pathinfo` return assoc without `extension`? :thinking: 
https://www.php.net/manual/en/function.pathinfo.php#refsect1-function.pathinfo-returnvalues

Reply:
yes it is possible. can intercept request and remove extension and pathinfo assoc array won't have key. changed to
```suggestion
        $fileExtension = pathinfo($input['name'])['extension'] ?? null;
```